server/src/socket/socket.js

```diff
@@ -182,8 +183,8 @@ export default (io) => {
   const fireGame = (player, socket) => {
     let game;
     if (gamesNeedingPlayers.length <= 0) {
-      gameID += 1;
-      const gameIDStr = gameID.toString();
+      // gameID += 1;
```

Comment:
Please remove this commented code at line 186.

Reply:
ok